<launch>
<<<<<<< HEAD
    <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui">
        <remap from="joint_states" to="champ/cmd_joints/relay" />
=======
    <node name="joints_calibrator_relay" pkg="champ_bringup" type="joint_calibrator_relay.py" output="screen" />

    <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui">
        <remap from="joint_states" to="joints_calibrator" />
>>>>>>> champ
    </node>
</launch>